guardian/newsletters-nx
# Fix build Issues & control email toggle via parameter store config

## What does this change?

- Corrects build issue by moving to v3 SSM client. 
- Updates service file to ensure logs written to journal
- Corrects Code domain
- Uses parameter store to configure email service toggle

## How to test

Tested in code - should now successfully build and email.
Best tested in code - created and publish a newsletter and verify emails are received. 



## How can we measure success?

Application running, Logs logging, emails sending

## Have we considered potential risks?

Think so - All config already set though we will need to determine appropriate recipients. Email service enable in CODE but not PROD so safe to merge
